CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 2.16.0
```

Comment:
This is the first time I've noticed the the VS code extension lists most recent work at the top rather than the bottom of the file. :) We should consider this for other repos.

Reply:
Yes! I'm going to raise PRs on other repos to fix this. I'll separate them from all other PRs.

---
In reply to: [325717773](https://github.com/microsoft/sarif-vscode-extension/pull/201#discussion_r325717773) [](ancestors = 325717773)